cmd/kpod/login.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"golang.org/x/crypto/ssh/terminal"
+
+	"github.com/containers/image/pkg/authentication"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+const defaultServer = "https://registry-1.docker.io"
```

Comment:
@rhatdan 's call, but this feels like it should be a openshift/rhel registry by default.  Maybe not now, but at some point.

Reply:
I don't like it either but it's a pretty strong user expectation right now. If we do choose a different default we need to make sure 'https://registry-1.docker.io' is easy to find in CLI help and man page since the registry string is not obvious.

Reply:
It should be in a configuration file /etc/containers/registries.conf
@baude Has been working onthis.

Reply:
User should be able to specify registry to login to, and he should be allowed to execute the command for each registry.